src/context.h

```diff
@@ -144,13 +150,16 @@ namespace librealsense
                                const std::map<std::string, std::weak_ptr<device_info>>& new_playback_devices);
         void raise_devices_changed(const std::vector<rs2_device_info>& removed, const std::vector<rs2_device_info>& added);
         void start_device_watcher();
-        void start_dds_device_watcher();
         std::shared_ptr<platform::backend> _backend;
         std::shared_ptr<platform::device_watcher> _device_watcher;
 
         std::map<std::string, std::weak_ptr<device_info>> _playback_devices;
         std::map<uint64_t, devices_changed_callback_ptr> _devices_changed_callbacks;
-        std::shared_ptr<librealsense::platform::device_watcher> _dds_watcher;
```

Comment:
This saved most of the
```
#ifdef BUILD_WITH_DDS
```

at the cpp file, don't you prefer it this way?

Reply:
The problem was that you have a pointer to `device_watcher` and I have a pointer to `dds_device_watcher` and I didn't have the type defined.

We can put it back later... let's see where we end up.

Reply:
👍